.github/workflows/core_build.yml

```diff
@@ -0,0 +1,59 @@
+name: core-build
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
```

Comment:
what would change if we use `fail-fast`? I guess it would be good for a contributor to see if the PR fails asap, but maybe I'm missing something here

Reply:
fail-fast actually aborts all the jobs if one fails.
So all would basically show a failure if single one fails.
https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
ok, then let's keep it to `false`